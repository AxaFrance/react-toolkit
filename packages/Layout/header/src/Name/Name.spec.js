import React from 'react';
import renderer from 'react-test-renderer';
import { Name } from './index';

describe('<Name>', () => {
  it('renders Name correctly', () => {
    const tree = renderer
      .create(
        <Name
          title="Nom de l&apos;application"
          subtitle="Baseline"
          img="test.svg"
          alt="Logo React"
          onClick={() => null}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
