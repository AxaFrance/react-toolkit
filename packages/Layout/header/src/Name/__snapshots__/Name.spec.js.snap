// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Name> renders Name correctly 1`] = `
<div
  className="af-header__name"
>
  <a
    className="af-logo"
    href="/"
    onClick={[Function]}
  >
    <img
      alt="Logo React"
      className="af-logo__brand"
      src="test.svg"
    />
  </a>
  <h2
    className="af-header__title"
  >
    Nom de l'application
    <span
      className="af-header__subtitle"
    >
      Baseline
    </span>
  </h2>
</div>
`;
